### PR TITLE
[QPE-564] test setup for components and some initial tests

### PR DESCRIPTION
**Pull request by kristofferlind**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -6,37 +7,72 @@ module.exports = (config) => {
...
+          '--disable-gpu',
+          '--disable-translate',
+          '--disable-extensions'
+        ]
       }
     },
     files: [
-      { pattern: 'src/*.spec.js', watched: false }
+      {
+        pattern: 'src/*.spec.js',
```
Comment (muenva) [inline karma.conf.js]: wouldn't this be duplicate of the one 👇 I thought that globbing with `**/*`means any level from 0 to any.

File: karma.conf.js [hunk 2]
```diff
@@ -6,37 +7,72 @@ module.exports = (config) => {
...
         ]
+      },
+      resolve: {
+        alias: {
+          'test-utilities': path.resolve('test/test-utilities')
+        },
+        // extensions: [
+        //   'js',
+        //   'jsx'
+        // ],
```
Comment (muenva) [inline karma.conf.js]: please remove the commented out code

File: karma.conf.js [hunk 3]
```diff
@@ -6,37 +7,72 @@ module.exports = (config) => {
...
+      }
     ],
     frameworks: ['jasmine'],
     preprocessors: {
-      'src/*.spec.js': ['webpack', 'sourcemap']
+      'src/**/*.spec.{js, jsx}': [
+        'webpack',
+        'sourcemap'
+      ],
+      'src/*.spec.{js, jsx}': [
```
Comment (muenva) [inline karma.conf.js]: same as ☝️

File: src/paint.jsx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 import $ from 'jquery';
 import initializeStore from './store';
 import React from 'react';
+// import ReactDOM from 'react-dom';
```
Comment (ahmed-Bazzara) [inline src/paint.jsx]: could you remove this :))

**Comment by ahmed-Bazzara** [comment]
merge conflicts

**Pull request merged**